RHDEVDOCS-3111: Update monitoring flows and capture customizing metrics (#48342)
openshift/openshift-docs: monitoring/reviewing-monitoring-dashboards.adoc

.Example dashboard in the Administrator perspective
image::monitoring-dashboard-administrator.png[]

> In the *Developer* perspective, you can access Kubernetes compute resources dashboards that provide the following statistics for a selected project:

Reviewer: ```suggestion
Use the *Developer* perspective to access Kubernetes compute resources dashboards that provide the following application metrics for a selected project:
```

Trying to be consistent with the usage of words.

Reviewer: Suggestion for L11: 
Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following <items/components>:

Suggest using a complete sentence for the lead-in sentence to reduce ambiguity and support translation. Refer to: https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences

Author: This seems to be a better lead-in sentence, "Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following items:"

Cc @vikram-raj
